systemtest/approvals/TestOTLPGRPCTraces.approved.json

```diff
@@ -10,7 +10,7 @@
             "opentelemetry/go"
         ],
         "agent.version": [
-            "1.25.0"
```

Comment:
[**For reviewers**] This change is due to upgrade of the otel-agent in the system test from `v1.25.0` to `v1.27.0`